File: templates/tpl.py

import base64
import os

# List of base64 chunks
chunks = [
{chunks}
]

# Join chunks and decode
b64 = "".join(chunks)
payload = base64.b64decode(b64)

# Write decoded bytes to file
with open("{filename}", "wb") as f:
    f.write(payload)

print(f"Wrote {{len(payload)}} bytes to {filename}")

File: templates/tpl.js
const fs = require("fs");
const path = require("path");

// Array of base64 chunks
const chunks = [{ chunks }];

// Join chunks and decode from base64
const b64 = chunks.join("");
const payload = Buffer.from(b64, "base64");

// Resolve output path and write file
const dest = path.resolve("{filename}");
fs.writeFileSync(dest, payload);

console.log("Wrote " + payload.length + " bytes to " + dest);
